extensions/amp-fx-collection/0.1/test/integration/test-amp-fx-fade-in-scroll.js

```diff
@@ -59,29 +60,29 @@ config.run('amp-fx-collection', {}, function () {
         expect(getOpacity(win)).to.equal(0);
         win.scrollTo(0, 0.1 * getViewportHeight(win));
         return Promise.resolve()
-          .then(timeout(2000))
+          .then(timeout(ANIMATION_TIME))
```

Comment:
Curious about why increasing the timeout causes the tests to run faster. Adding @alanorozco for review.

Reply:
It isn't an upper limit, it is just waiting `2000ms` for the transition to play. However, the transition time has been overridden in the template. This might also be achievable by rewriting with `window.requestAnimationFrame` but I wanted to take the path of least change.

Reply:
Aha, that explains it.